backend/api/routers/moodle.py

```diff
@@ -89,16 +90,38 @@ def create_course_build(
         "instructor_firstname": instructor_firstname,
         "instructor_lastname": instructor_lastname,
         "instructor_email": instructor_email,
-        "school_district": school_district_name,
+        "school_district_name": school_district_name,
         "academic_year": academic_year,
         "academic_year_short": academic_year_short,
         "course_name": course_name,
         "course_shortname": maybe_course_shortname,
-        "creator": current_user["email"],
+        "creator_email": current_user["email"],
         "status": status,
     }
 
 
+@router.get("/moodle/course/build", dependencies=[Depends(verify_user)])
+def get_course_builds(
+    db: Session = Depends(database.get_db),
+    academic_year: str = None,
+    instructor_email: str = None,
+) -> list[GetCourseBuildSettings]:
+    course_build = database.get_course_build(db, academic_year, instructor_email)
+    for course in course_build:
+        if course.school_district_id:
+            school_district_name = course.school_district.name
+            course.school_district_id = school_district_name
+
+        if course.creator_id:
+            creator_email = course.creator.email
+            course.creator_id = creator_email
+
+        if course.status:
+            course.status = course.status.value
```

Comment:
I don't think these conditional checks are necessary since the schema requires them to exist. For example, would the following be problematic?

```python
    for course in course_build:
        course.school_district_id = course.school_district.name
        course.creator_id = course.creator.email
        course.status = course.status.value
```

Reply:
You are correct. The conditionals were left over from when I was going to apply a different concept.